dojo/finding/views.py

```diff
@@ -665,11 +666,13 @@ def edit_finding(request, fid):
 
             # Push to Jira?
             push_to_jira = False
-            if enabled:
+            if push_all_issues_enabled:
                 push_to_jira = True
-            elif 'jiraform-push_to_jira' in request.POST:
-                jform = JIRAFindingForm(request.POST, prefix='jiraform', enabled=enabled)
+            # elif 'jiraform-push_to_jira' in request.POST:
+            elif request.POST['jiraform-push_to_jira']:
+                jform = JIRAFindingForm(request.POST['jiraform-push_to_jira'], prefix='jiraform', enabled=True)
                 if jform.is_valid():
+                    # push_to_jira = True  ## easy way out but why?
                     push_to_jira = jform.cleaned_data.get('push_to_jira')
```

Comment:
Any idea why this always returns `False` even if the checkbox is ticked?

Reply:
Would you need to replace `request.POST['jiraform-push_to_jira']` by `request.POST`?

Reply:
oh I see on slack you already solved it?

Reply:
No. Either one, result seems to be the same.